src/test/integration/FileTestModule.groovy

```diff
@@ -25,7 +24,7 @@ public class FileTestModule extends AbstractModule {
         dataCollectorBinder.addBinding().to(StatisticsCollector.class)
         dataCollectorBinder.addBinding().to(BuildRequester.class)
 
-        bind(CommitFilter.class).to(InCommitListMutuallyModifiedMethodsTextualConflictCommitFilter.class)
+        bind(CommitFilter.class).to(InCommitListMutuallyModifiedMethodsFilter.class)
```

Comment:
@rafaelmotaalves OK, now I see. but better organizing the commits in a branch in your local repository can make code review easier.